### PR TITLE
Add tagging documentation

### PR DESCRIPTION
**Pull request by marcraminv**

### CODE REVIEW

File: docs/tagging.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+id: tagging
+title: Tagging
+---
+
+Weaver brings some default tags as `ignore` or `cancel` to disable the tests temporally.
```
Comment (Baccata) [inline docs/tagging.md:6]:
```suggestion
Weaver provides some constructs to dynamically tag tests as `ignored` or `cancelled` :   
```

Comment (marcraminv) [reply docs/tagging.md]: thanks for feedback @Baccata !

File: docs/tagging.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+object MySuite extends SimpleIOSuite {
+
+
+  simpleTest("hello tag ignore") {
+    for {
+      x <- IO.delay(1)
+      y <- IO.delay(2)
+    } yield expect(x == y)
+    ignore("I want to ignore this test because is flaky")
```
Comment (Baccata) [inline docs/tagging.md:20]: though this works, the for-comprehension is not monadically tied with the `ignore` call. A more relevant example would be something like : 

```scala
import cats.implicits._ 

simpleTest("only on CI") {
    for {
      onCI <- IO(sys.env.get("CI").isDefined)
      _    <- ignore("not on CI").unlessA(onCI)
      x    <- IO.delay(1)
      y    <- IO.delay(2)
    } yield expect(x == y)
}
```

Same goes for cancel (if you want to vary the example, you could cancel the test when it's the morning or something)

Comment (marcraminv) [reply docs/tagging.md:20]: Right, has sense inside the for-comprehension, I change it. 👍

**Review by Baccata — APPROVED** [review]
Thanks for contributing 😄

**Pull request merged**